src/gcs/gcsDrive.ts

```diff
@@ -54,19 +55,37 @@ export class GCSDrive implements Contents.IDrive {
    * @returns IModel directory containing all the GCS buckets for the current project.
    */
   private async getBuckets() {
+    let paragraph: HTMLElement | null;
     let searchInput = document.getElementById('filter-buckets-objects');
     //@ts-ignore
     let searchValue = searchInput.value;
     const content = await GcsService.listBuckets({
       prefix: searchValue
     });
+
+    if (content?.error?.code) {
+      if(document.getElementById('gcs-list-bucket-error')){
+        document.getElementById('gcs-list-bucket-error')?.remove()
+      }
+      const para = document.createElement('p');
+      para.id = 'gcs-list-bucket-error';
+      para.style.color = '#ff0000';
+      para.innerHTML = content?.error?.message;
```

Comment:
don't use innerHTML, use textContent instead: https://www.dhairyashah.dev/posts/why-innerhtml-is-a-bad-idea-and-how-to-avoid-it/